.github/workflows/ci.yml

```
@@ -87,7 +87,11 @@ jobs:
         run: |
           docker push "$CORE_CI_IMAGE:latest"
           docker push "$CORE_CI_IMAGE:ci--$BRANCH_REF"
-      - name: Run Rubocop linting
+      - name: Run Python flake8 linting
+        run: |
```

Currently this will run for _every_ step in the matrix (18 times), I think we could limit it to only the python build by doing:

```suggestion
        if:  matrix.suite == 'python'
        run: |
```